cabal-install/src/Distribution/Client/Setup.hs

```diff
@@ -2354,18 +2354,18 @@ userConfigCommand = CommandUI {
   commandDescription  = Just $ \_ -> wrapText $
        "When upgrading cabal, the set of configuration keys and their default"
     ++ " values may change. This command provides means to merge the existing"
-    ++ " config in ~/.cabal/config"
+    ++ " config in ~/.config/cabal/config"
```

Comment:
Same concern about explicit paths here and in the rest of this message (three more times).